code/game/gamemodes/events/holidays/Holidays.dm

```diff
@@ -31,36 +31,33 @@ var/global/Holiday = null
 		if(1)	//Jan
 			switch(DD)
 				if(1)							Holiday = "New Year's Day"
+				if(10)							Holiday = "Houseplant Appreciation Day"
+				if(25)							Holiday = "Opposite Day"
 
 		if(2)	//Feb
 			switch(DD)
 				if(2)							Holiday = "Groundhog Day"
 				if(14)							Holiday = "Valentine's Day"
 				if(17)							Holiday = "Random Acts of Kindness Day"
+				if(29)
+					if(YY == (20 || 24 || 28))	Holiday = "Leap Day"

```

Comment:
Is the year check even necessary? `DD == 29` should only happen on leap days anyways, so is there a need to check the year?